packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -217,12 +211,13 @@ const makeAuctionDriver = async (t, customTerms, params = defaultParams) => {
     const pa = makeManualPriceAuthority({
       actualBrandIn: collateralBrand,
       actualBrandOut: currency.brand,
-      timer: timerService,
+      timer,
       initialPrice: makeRatio(100n, currency.brand, 100n, collateralBrand),
     });
     priceAuthorities.init(collateralBrand, pa);
     registry.registerPriceAuthority(pa, collateralBrand, currency.brand);
 
+    // @ts-expect-error ts seems confused here.
```

Comment:
let's get to the bottom of this before merge

Reply:
I spent a while looking for solutions, and didn't find anything. Do you have time to look together?

Reply:
Thanks for your help. fixed.